scripts/install.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+CLI_HOME=~/.vrap-cli
+LIB_FOLDER=$CLI_HOME/lib
+JAR_FILE_PATH=$LIB_FOLDER/codegen-cli-${VRAP_VERSION}.jar
```

Comment:
You should set a default value for VRAP_VERSION, so that only if you want to have a specific version there is a need to set this value

https://unix.stackexchange.com/a/122848

Reply:
Actually the risk here is that someone might depend on it for a build tool and once we change the version it would fail, thus i would prefer to force him to specify a specific version.

Reply:
I didn't say that it should not be possible to override it, only that there is no need to specify it. Just makes it more convenient.